addons/hr_holidays_attendance/models/hr_leave_accrual_plan_level.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class AccrualPlanLevel(models.Model):
+    _inherit = "hr.leave.accrual.level"
+
+    frequency_hourly_source = fields.Selection(
+        selection=[
+            ('calendar', 'Calendar'),
+            ('attendance', 'Attendances')
+        ],
+        default='calendar',
+        compute='_compute_frequency_hourly_source',
```

Comment:
As we want to ensure that `frequency_hourly_source` is set to calendar if accrual is set at start, shouldn't we have an inverse setting `accrued_gain_time` to 'end' if we set attendance?